monsteer/plugin/spikeReport.cpp

```diff
@@ -20,76 +19,78 @@
 
 #include "spikeReport.h"
 
-#include <zeroeq/zeroeq.h>
-
 #include <lunchbox/clock.h>
 #include <lunchbox/pluginRegisterer.h>
 #include <lunchbox/uri.h>
 
 #include <brion/version.h>
 
-extern "C" int LunchboxPluginGetVersion() { return BRION_VERSION_ABI; }
+#include <zeroeq/zeroeq.h>
+
+extern "C" int LunchboxPluginGetVersion()
+{
+    return BRION_VERSION_ABI;
+}
 extern "C" bool LunchboxPluginRegister()
 {
     lunchbox::PluginRegisterer< monsteer::plugin::SpikeReport > registerer;
     return true;
 }
 
+#define RECEIVE_TIMEOUT 100 // ms
```

Comment:
This is a legacy code that I didn't really touched. I don't know why it is there in this form.